File: src/main/java/com/p6/paymybuddy/Service/Data/TransactionInternal.java
package com.p6.paymybuddy.Service.Data;

import lombok.Data;

@Data
public class TransactionInternal {
    private Long id;
    private String description;
    private Double amount;
    private String timeTransaction;
    private Person crediteur;
    private Person debiteur;

    public TransactionInternal() {
    }

    
}

File: frontEndAngular/src/app/transaction-internal/transactionInternal.ts
import {Person} from "../person/Person";

export interface TransactionInternal {
  idTransaction: number;
  description: string;
  amount: string;
  timeTransaction: string;
  crediteur : number ;
  debiteur: number;
  emailLogin:string;
}

export interface TransactionInternalRequest {
  idTransaction: number;
  description: string;
  amount: string;
  timeTransaction: string;
  crediteur : Person ;
  debiteur: Person;
  emailLogin:string;
}

